.github/workflows/ci.yml

```diff
@@ -14,16 +14,18 @@ concurrency:
 
 jobs:
   static:
-    uses: dvsa/.github/.github/workflows/php-library-static.yml@v3.2.3
+    uses: dvsa/.github/.github/workflows/php-library-static.yml@v4.0.1
```

Comment:
I'm thinking for the CI tasks do we want to run the workflows not for 'library'.  And run these all in prefer-stable mode?

Reply:
tbh, I don't know the answer to the first question. I'm cool to go with prefer-stable, but I can't remember where it's set. I saw the `matrix` config you shared yesterday, but I don't know where it is.

Reply:
Changed to the non-library versions. I've also bumped the security workflow after it failed at v3.2.3. Unfortunately 4.0.1 also seems to be failing, so I'm going to roll that back now.

Reply:
What is the error with the latest version of the security action, we just recently released a version of dvsa github v4.1.0 (added a composer-version parameter to php libraries), does this work?

Reply:
It's actually doing the same thing on @alexisc-kainos 's PR, which is on 4.1.0, so I've not bumped anything. I presume that someone has switched something on recently as this wasn't appearing a few days ago.